src/stores/epics/roadEpic.ts

```diff
@@ -29,4 +29,24 @@ export const getRoadEpic: Epic = (action$) =>
     })
   );
 
+export const removeRoadEpic: Epic = (action$) =>
+  action$.pipe(
+    ofType(ActionTypes.REQUEST_REMOVE_ROAD),
+    mergeMap((action) => {
+      const { roadId } = action.payload;
+      return from(roadAPI.removeRoad({ roadId })).pipe(
+        map((payload) => ({
+          type: ActionTypes.REQUEST_GET_ROAD_SUCCESS,
```

Comment:
```suggestion
          type: ActionTypes.REQUEST_REMOVE_ROAD_SUCCESS,
```
아래 ERROR부분도 같이 바꿔줘야 할거같아용!